src/platform/forms/save-in-progress/actions.js

```diff
@@ -251,12 +252,14 @@ export function fetchInProgressForm(
   //  redux store, but form.migrations doesn’t exist (nor should it, really)
   return (dispatch, getState) => {
     const trackingPrefix = getState().form.trackingPrefix;
+    const apiUrl =
+      VA_FORM_IDS_IN_PROGRESS_FORMS_API[formId] || '/v0/in_progress_forms/';
```

Comment:
How do you feel about standardizing this lookup in a function which does just this? Then we can do something like

```js
const apiUrl = inProgressApi(formId);
```

It'll then consolidate the `/v0/in_progress_forms/` magic string to one place.

Reply:
Sounds like a good idea!